website/source/docs/builders/oracle-bmcs.html.md

```diff
@@ -0,0 +1,142 @@
+    [BMCS config file](https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/sdkconfig.htm)
+    to use. Defaults to `DEFAULT`.
+
+ -  `availability_domain` (string) - The name of the
+    [Availability Domain](https://docs.us-phoenix-1.oraclecloud.com/Content/General/Concepts/regions.htm)
+    within which a new instance is launched and provisioned.
+    The names of the Availability Domains have a prefix that is specific to
+    your [tenancy](https://docs.us-phoenix-1.oraclecloud.com/Content/GSG/Concepts/concepts.htm#two).
+
+    To get a list of the Availability Domains, use the
+    [ListAvailabilityDomains](https://docs.us-phoenix-1.oraclecloud.com/api/#/en/identity/latest/AvailabilityDomain/ListAvailabilityDomains)
+    operation, which is available in the IAM Service API.
+
+ -  `base_image_ocid` (string) - The OCID of the
+    [Oracle-provided base image](https://docs.us-phoenix-1.oraclecloud.com/Content/Compute/References/images.htm)
+    to use. This is the unique identifier of the image that will be used to
+    launch a new instance and provision it.
+
+    To get a list of the accepted image OCIDs, use the
+    [ListImages](https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/latest/Image/ListImages)
+    operation available in the Core Services API.
+
+ -  `compartment_ocid` (string) - The OCID of the
+    [compartment](https://docs.us-phoenix-1.oraclecloud.com/Content/GSG/Tasks/choosingcompartments.htm)
+
+ -  `fingerprint` (string) - Fingerprint for the key pair being used. Overrides
+    value provided by the
+    [BMCS config file](https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/sdkconfig.htm)
+    if present.
+
+ -  `image_name` (string) - The name to assign to the resulting custom image.
+
+ -  `key_file` (string) - Full path and filename of the private key. Overrides
+    value provided by the
+    [BMCS config file](https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/sdkconfig.htm)
+    if present.
+
+ -  `pass_phrase` (string) - Pass phrase used to decrypt the private key used
+    to sign requests to the BMCS API. Overrides value provided by the
+    [BMCS config file](https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/sdkconfig.htm)
+    if present.
+
+ -  `region` (string) - An Oracle Bare Metal Cloud Services region. Overrides
+    value provided by the
+    [BMCS config file](https://docs.us-phoenix-1.oraclecloud.com/Content/API/Concepts/sdkconfig.htm)
+    if present.
+
+ -  `shape` (string) - The template that determines the number of
+    CPUs, amount of memory, and other resources allocated to a newly created
+    instance.
+
+    To get a list of the available shapes, use the
+    [ListShapes](https://docs.us-phoenix-1.oraclecloud.com/api/#/en/iaas/20160918/Shape/ListShapes)
+    operation available in the Core Services API.
+
+ -  `subnet_ocid` (string) - The name of the subnet within which a new instance
```

Comment:
May wish to note that the subnet will need to be configured to allow tcp/22 for the ssh communicator. (It is true on the default subnet bmcs creates but isn't for other subnets, and may not be obvious this has to be done.)